CHANGELOG.rst

```diff
@@ -7,6 +7,16 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 2.2.0 (2024-07-23)
+--------------------------
+
+* Added: ``audeer.script_dir()``
+  which returns the directory of the file,
+  in which the function is called,
+  or the current working directory
+  when called in an interactive session
```

Comment:
Having no full stop at the end of a sentence is more common when it comes to CHANGELOG entries.